C/manual/ch_Finance-Quote.xml

```diff
@@ -165,6 +154,13 @@ SYNOPSIS
       documentation is displayed, you will have to continue with this chapter.
     </para>
 
+    <note>
+      <para>In &app; version 5.0 the old commands <userinput>gnc-fq-check</userinput>,
+        <userinput>gnc-fq-dump</userinput> and <userinput>gnc-fq-helper</userinput> have been
+        removed and replaced with similar functions in &app-cli;.
+      </para>
+    </note>
+
```

Comment:
This should be the other way around:
```
<para>In &app; version 4.x and earlier one used separate perl programs <userinput>gnc-fq-check</userinput> instead of <userinput>&app-cli; -Q info</userinput> and <userinput>gnc-fq-dump</userinput> instead of <userinput>&app-cli; -Q dump</userinput>.